### DISCUSSION TITLE
OTA TASMOTIZER

### DISCUSSION THREAD

**Discussion by ratfink11** [tasmota/tasmotizer#135] [General]
Sorry all,  but this is just eating at me from a common sense standpoint.... 

So Im wondering why OTA install is not an option? Does everyone really think its fun and easier to solder header pins and make cables? Purchase Dupont connector crimpers? 

I mean, if the chinese gov can hack ewelink, sonoff devices,  is it really that hard to create and keep an updated OTA option on the table?

For obvious reasons,  or at least as I see it.  Is it really that much easier dissasembling a sonoff device, losing or stripping out screws, solder bridge a trace, cut a trace, look at wrong, lightning strikes, waving a dead chicken, .....

Having to actually FIGURE OUT which hardware usb to esxxx, serial, parralel, batman, robin, 18 inch floppy..... because ALL the instructions to flash are providing software steps.... not actually what hardware to purchase..... yeah,  they all mention the little fartsy usb device.... but leave out the fact you need header pins, soldering iron, and JUMPER WIRE..... At the very least pins in jumper wire, while  taking your 3rd hand to hold the 4th wire to correct pins while clicking the mouse button with your nose?

Or,  am I making too much sense and im clueless?

This is a great tool,  just cant figure out why everyone gave up on the OTA Option?

**Comment by jziolkowski** [comment]
But no one gave up on OTA. It's just that tasmotizer is a tool used for serial flashing if that's your choice. Not to mention that in most scenarios, using flash write is the most reliable. It ensures that all firmware regions are properly cleared before flashing new image. It's resilient to wifi issues etc. 

Also, not all manufacturers make it easy to flash alternative firmware via the existing ota solutions. For example tuya made efforts to secure their ota protocol when the community discovered how to (ab)use it to flash alternative firmwares without soldering.

Your question is analogous to "why create tools related to ethernet when we have WiFi". It's different solutions to common problems. One doesn't rule out the other.

**Comment by ratfink11** [comment]
I understand what Tasmotizer is and its function.

As far as finding or locating detailed instructions, or even clearer instructions on OTA for Tasmota I havent had much luck? Are they calling it Web browser updates now? Like ufos are now called UAPs? Im cool looking like an idiot, but any info I find is dated 2017,  then after reading and searching for 3 hours the fine print says it no longer works with Win10 because fish get thirsty in warmer waters....

...not sure about Tuya, since I never brought them up, dont know who they are, never heard of them, dont use them.... In any case any company that apparently is going to the trouble of epoxying thier cases shut, updating firmware to prevent firmware updates, making it painfully obvious they no longer want to be in business.... There are many many other sensor options out thier... Shellys for one.... They must be one of those companies that shake the trees to make the wind blow? Yeah, that and Morgan Fairchild, yeah  thats the ticket....

Seriously and sincerely I appreciate your feedback and info,  so far it appears to be what I have deduced....Thankyou.


________________________________
